enterprise/backend/src/metabase_enterprise/metabot_v3/client.clj

```diff
@@ -153,6 +156,25 @@
                       {}
                       e)))))
 
+(defn find-outliers-request
+  "Make a request to AI Service to find outliers"
+  [values]
+  (try
+    (let [url (find-outliers-endpoint-url)
+          body {:values values}
+          options (build-request-options body)
+          response (post! url options)]
+      (if (= (:status response) 200)
+        (u/prog1 (decode-response-body (:body response))
+          (log/debugf "Response (decoded):\n%s" (u/pprint-to-str <>)))
+        (throw (ex-info (format "Error: unexpected status code: %d %s" (:status response) (:reason-phrase response))
```

Comment:
**suggestion (not-blocking):** There can also be a 400 response if we sent too many values (at the moment the limit is 500k though :D)